draft-ietf-quic-transport.md

```diff
@@ -2345,7 +2345,7 @@ following layout:
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|0|1|               Unpredictable Bits (182..)                ...
+|0|1|               Unpredictable Bits (422..)                ...
```

Comment:
This one remains particularly thorny.  Now there is no way to generate a stateless reset that is a reasonable size.  Yes, this is technically bounded, but now the naive stateless reset is considerably bigger than the packet that triggers it in many more cases.

Reply:
Isn't there a rule that you can't send a reset that's biger than the originating packet?

Reply:
eternal ping pong rule

Reply:
you can get around sending a reset only once per every 4 received packets on average, but consensus dislike random.

Reply:
discussed in London; I think we're OK with this